libs/mock-block-dock/src/service-module-callbacks.ts

```diff
@@ -0,0 +1,113 @@
+import { EmbedderServiceMessageCallbacks } from "@blockprotocol/service";
+import axios from "axios";
+
+const bpApiBaseURL = process.env.BP_API_BASE_URL ?? "http://localhost:3000/api";
```

Comment:
I'm not sure about `localhost` as the default – most people developing blocks will not also have the `blockprotocol` repo and site running. I think we should default to supporting people who can just use the production API instance (that'll be fine for us most of the time too, unless we want to debug the API itself). I've also suggested a tweak to the environment variable as we use `BLOCK_PROTOCOL_SITE_HOST` elsewhere in the repo (in the publish script)

```suggestion
const bpApiBaseURL = `${process.env.BLOCK_PROTOCOL_SITE_HOST ?? "http://localhost:3000}/api`;
```

Reply:
That's a good point, I've modified the default value to be `blockprotocol.org` and made use of `BLOCK_PROTOCOL_SITE_HOST` in https://github.com/blockprotocol/blockprotocol/pull/966/commits/83aea17a2ea8d7c07a1dae77d0c54e7d97235be5